package.json

```diff
@@ -10,7 +10,8 @@
   },
   "devDependencies": {
     "eslint": "3",
-    "mocha": "3"
+    "mocha": "3",
+    "eslint-plugin-mocha": "4.9.0"
```

Comment:
@seanmonstar i am working on testing now. didn't work on one of my machines, so i must try it on another....